### PR TITLE
Proposal: add sample for R lang

### PR DESCRIPTION
**Pull request by mrchypark**
add sample functions for R language with alpine image.

## Description

r-base official image repository does not support alpine image but recommend other repository. please see below issue.
<https://github.com/rocker-org/rocker/issues/231>

So i use build that repo dockerfile for R language alpine.

And add r lang sameple functions for faas. 

## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/alexellis/faas/blob/master/CONTRIBUTING.md))

<https://github.com/alexellis/faas/issues/152>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.    
- [] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by mrchypark** [comment]
How do I check signed-off commit? And i add sample functions for r language. How do I test that?

**Comment by alexellis** [comment]
Hi @mrchypark - I saw you figured out the sign off for the agreement in the [CONTRIBUTING guide](https://github.com/alexellis/faas/blob/master/CONTRIBUTING.md). We would be really interested to see an example of running some R statistics for possible inclusion in our community collection of functions: https://github.com/faas-and-furious/

File: sample-functions/BaseFunctions/R/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM artemklevtsov/r-alpine:latest
+
+ADD https://github.com/alexellis/faas/releases/download/0.5.1-alpha/fwatchdog /usr/bin
```
Comment (alexellis) [inline sample-functions/BaseFunctions/R/Dockerfile]: It would be great if you could change `0.5.1-alpha` to `0.6.2` (without `alpha`).

Comment (mrchypark) [reply sample-functions/BaseFunctions/R/Dockerfile]: Ok. I changed

**Comment by mrchypark** [comment]
@alexellis Thank you for your attention. I'll try to add readme for sample base function for R language.
[This repo](https://github.com/faas-and-furious/) is like function collections for faas? If so, I'm interested in add examples using r. I'm trying to figure out good examples.

**Comment by mrchypark** [comment]
- [x] Add a README file inside the root folder of my new example showing how to build/deploy and invoke your R function.

**Comment by alexellis** [comment]
Hi @mrchypark this is really shaping up now. I noticed we've got 16 commits for this small change and not all of them are signed-off. If you squash the commits into one then sign it off we can get this merged. 👍 

If you need help with the squashing let me know (it has to be done on your account though). 

Once we've merged this it would be very cool to try creating some graphics or simulations from a dataset.

**Comment by mrchypark** [comment]
@alexellis  Thank you for your feedback. I'm not used to git. I'm trying to do what your saying and notice you. 👍

**Comment by mrchypark** [comment]
@alexellis How about this time?

**Comment by alexellis** [comment]
Thank you for perservering with this 👍 Merged.

**Pull request merged**